kgl-glfw/src/jvmMain/kotlin/com/kgl/glfw/Window.kt

```diff
@@ -267,127 +267,183 @@ actual class Window @PublishedApi internal constructor(val ptr: Long) : Closeabl
 
 	actual fun getMouseButton(button: MouseButton): Action = Action.from(glfwGetMouseButton(ptr, button.value))
 
-	actual fun setPosCallback(callback: WindowPosCallback?) {
+	private var windowPosCallback: WindowPosCallback? = null
```

Comment:
I've been avoiding doing this (keeping a reference to the callbacks) on the JVM target but it looks like there isn't much choice anymore.
If we're going to keep references now, then I think we should use them like is done on the Native target.
Essentially, copy over the `setCallback` function from the native side here and use it the same way.

```
val context = glfwGetWindowUserPointer(window)!!.asStableRef<Window>().get()
context.windowPosCallback?.invoke(context, width, height)
```

`glfwGetWindowUserPointer(window)!!.asStableRef<Window>().get()` on the JVM looks like this https://github.com/Dominaezzz/kgl/blob/8f1f19b64f1a54745d15755d7ae247273650ed58/kgl-glfw/src/jvmMain/kotlin/com/kgl/glfw/Glfw.kt#L34